.github/integration/tests/postgres/20_inbox_queries.sh

```diff
@@ -15,4 +15,6 @@ for host in migrate postgres; do
         echo "Mark file as uploaded failed"
         exit 1
     fi
-done
\ No newline at end of file
+done
+
+echo "20_inbox_queries completed successfully"
```

Comment:
Since it refers to the rootname of the script, maybe better to use 
```bash
scriptname=$(basename "$0")
rootname=${scriptname%.*}
echo "${rootname} completed successfully"
```
instead of changing the name in every script.

Reply:
Don't over complicate things